web/html/src/manager/admin/payg-shared/sshData/payg-ssh-data-form.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from "react";
+
+import { paygProperties } from "manager/admin/payg/payg";
+
+import { Form, Text } from "components/input";
+import { InputBase } from "components/input/InputBase";
+
+type PropsForm = {
+  paygSshData: paygProperties;
+  errors: any;
+  onChange: Function;
+  isInstance: boolean;
+  editing?: boolean;
+};
+
+const PaygSshDataForm = (props: PropsForm) => {
+  return (
+    <Form
+      model={props.paygSshData}
+      errors={props.errors}
+      onChange={(model) => {
+        props.onChange(model);
+      }}
+    >
+      <PaygSshDataFormFields paygSshData={props.paygSshData} isInstance={props.isInstance} editing={props.editing} />
+    </Form>
+  );
+};
+
+export default PaygSshDataForm;
+
+type PropsFields = {
+  paygSshData: paygProperties;
+  isInstance: boolean;
+  editing?: boolean;
+};
+
+export const PaygSshDataFormFields = (props: PropsFields) => {
+  let prefix = props.isInstance ? "" : "bastion_";
+  return (
+    <React.Fragment>
+      <div className="row">
+        <Text
+          required={props.isInstance ? true : false}
```

Comment:
Since `isInstance` is already boolean, this can be `required={props.isInstance}`. If this is not the case, the prop definition needs to be updated.